templates/default/src/app/pages/Data.tsx

```diff
@@ -1,16 +1,17 @@
+import React, { useState } from "react";
+
+import { Flex } from "@dynatrace/strato-components/layouts";
+import { Heading, Paragraph } from "@dynatrace/strato-components/typography";
 import {
-  DQLEditor,
-  QueryStateType,
   RunQueryButton,
-  TimeseriesChart,
-  convertToTimeseries,
-  recommendVisualizations,
-} from "@dynatrace/strato-components-preview";
-import { Flex, Paragraph, Heading } from "@dynatrace/strato-components";
-import { useDqlQuery } from "@dynatrace-sdk/react-hooks";
+  type QueryStateType,
+} from "@dynatrace/strato-components-preview/buttons";
+import { TimeseriesChart } from "@dynatrace/strato-components-preview/charts";
+import { convertToTimeseries } from "@dynatrace/strato-components-preview/conversion-utilities";
```

Comment:
Please keep the `recommendVisualizations` import else the template breaks.

Reply:
Thanks for spotting this! 🙌